src/parsing/parsers/generic/jison/delete/delete.test.ts

```diff
@@ -0,0 +1,65 @@
+import {ColumnSuggestion, KeywordSuggestion, parseGenericSql} from '../../../../index';
+import {describe, expect, it} from '@jest/globals';
+
+// TODO: reuse it in more places
+const SUGGEST_TABLES_VALUE = {};
+
+describe('delete statement', () => {
+    it("should not report errors on short delete statement", () => {
+        expect(parseGenericSql('DELETE FROM test_table; ', '').errors)
+            .toEqual(undefined);
+    })
+    it('should not report errors on full delete statement', () => {
+        expect(parseGenericSql('DELETE FROM test_table WHERE column != 1; ', '').errors)
+            .toEqual(undefined);
+    })
+    it('should suggest DELETE', () => {
+        const keyword: KeywordSuggestion = { value: 'DELETE', weight: -1 };
+        expect(parseGenericSql('', '').suggestKeywords)
+            .toContainEqual(keyword);
+    })
+    it('should suggest FROM', () => {
+        const keyword: KeywordSuggestion = { value: 'FROM', weight: -1 };
+        expect(parseGenericSql('DELETE ', '').suggestKeywords)
+            .toContainEqual(keyword);
+    })
+    it('should suggest tables', () => {
+        expect(parseGenericSql('DELETE FROM ', '').suggestTables)
+            .toEqual(SUGGEST_TABLES_VALUE);
+    })
+    it('should suggest WHERE', () => {
+        const keyword: KeywordSuggestion = { value: 'WHERE', weight: -1 };
+        expect(parseGenericSql('DELETE FROM test_table ', '').suggestKeywords)
```

Comment:
I'm scared that we didn't check whole result as we did it in `test.json`, imo its reducing significance of tests